middlewares/settings.middleware.js

```diff
@@ -19,18 +22,8 @@ const Settings = require("../models/settings.model");
  * @description Put relevent settings attributes into settingsDetails
  */
 function parsePatch(req, res, next) {
-    let settingsDetails = {};
-
-    for (const val in req.body) {
-        // use .hasOwnProperty instead of 'in' to get rid of inherited properties such as 'should'
-        if (Settings.schema.paths.hasOwnProperty(val)) {
-            settingsDetails[val] = req.body[val];
-            delete req.body[val];
-        }
-    }
-
-    req.body.settingsDetails = settingsDetails;
-    return next();
+    let parseSettingDetails = Services.ParsePatch.parsePatch(Model.Settings, "settingDetails");
```

Comment:
The model name is Settings, so I believe that the name of the variable should be `settingsDetails`, to follow the xDetails style.

Reply:
Yep thats whats causing the test failure, good catch I should have seen this earlier :/